doc/index.rst

```diff
@@ -154,6 +154,9 @@ Note that some of these currently tied only to a single option, but we still lea
 * `CATKIN_PARALLEL_JOBS` (default: -p4): Maximum number of packages to be built in parallel that is passed to underlining build tool. As of Jan 2016, this is only enabled with `catkin_tools`. See for more detail about `number of build jobs <http://catkin-tools.readthedocs.org/en/latest/verbs/catkin_build.html#controlling-the-number-of-build-jobs>`_ and `documentation of catkin_tools <https://catkin-tools.readthedocs.org/en/latest/verbs/catkin_build.html#full-command-line-interface>`_ that this env variable is passed to internally in `catkin-tools`.
 * `CATKIN_PARALLEL_TEST_JOBS` (default: -p4): Maximum number of packages which could be examined in parallel during the test run. If not set it's filled by `ROS_PARALLEL_JOBS`.
 * `CI_PARENT_DIR` (default: .ci_config): (NOT recommended to specify) This is the folder name that is used in downstream repositories in order to point to this repo.
+* `DOCKER_IMAGE` (default: not set): Selects a Docker images different from default one. Please note, this disables the handling of `ROS_REPOSITORY_PATH` and `ROS_DISTRO`.
```

Comment:
These 2 variables are kind of "default" (although not declared so [as of now](https://github.com/ros-industrial/industrial_ci/blob/d4bb8681bf02f5b69ba157ee2864218dc3c82693/doc/index.rst#variables-you-can-configure)), and it's not clear what will get tested without these variables working. Maybe we want to add a sample section for this and the subsequent 2 new `DOCKER_*` variables?